packages/api-reference/src/components/Content/Models/CollapsedModel.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { ScalarIcon } from '@scalar/components'
+import { nextTick, ref, watch } from 'vue'
+
+import { scrollToId } from '../../../helpers'
+import { useNavState } from '../../../hooks'
+import Anchor from '../../Anchor/Anchor.vue'
+import { Section } from '../../Section'
+import { Schema, SchemaHeading } from '../Schema'
+
+const props = defineProps<{ name: string; schemas: Record<string, any> }>()
+const { hash, getModelId } = useNavState()
+
+const showCollapsedItems = ref(false)
+
+watch(
```

Comment:
@amritk @hanspagel I'm not sure if we need to do anything here for path routing or SSR 🤔